src/abstractdataframe/abstractdataframe.jl

```diff
@@ -334,16 +334,20 @@ end
 Report descriptive statistics for a data frame
 
 ```julia
-describe(df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissing, :nunique, :eltype])
+describe(df::AbstractDataFrame)
+describe(df::AbstractDataFrame, stats::Union{Symbol, Pair{Symbol}}...)
 ```
 
 **Arguments**
 
-* `df` : the AbstractDataFrame
-* `stats::Union{Symbol,AbstractVector{Symbol}}` : the summary statistics to report. If
-  a vector, allowed fields are `:mean`, `:std`, `:min`, `:q25`, `:median`,
-  `:q75`, `:max`, `:eltype`, `:nunique`, `:first`, `:last`, and `:nmissing`. If set to
-  `:all`, all summary statistics are reported.
+* `df` : the `AbstractDataFrame`
+* stats::Union{Symbol, Pair{Symbol}}... : the summary statistics to report. 
+    * Arguments can be symbols from the following: `:mean`, `:std`, `:min`, `:q25`, 
```

Comment:
"Arguments can be" should be moved outside of the list as it applies to all bullet points. Then you can go straight to the point for each type.

Reply:
fixed.